flycheck.el

```diff
@@ -5640,7 +5640,10 @@ See URL `http://elixir-lang.org/'."
    (warning line-start (file-name) ":" line ": warning:" (message) line-end)
    ;; Warnings from older Elixir versions
    (warning line-start (file-name) ":" line ": " (message) line-end))
-  :modes elixir-mode)
+  :modes elixir-mode
+  :predicate
+  (lambda ()
+    (not (string-match "\.exs$" buffer-file-name))))
```

Comment:
ah, thanks. I'm still kind of new to emacs, so this is one of my first times actually diving into elisp beyond basic config stuff.